sdk/Trace/Span.php

```diff
@@ -334,12 +372,19 @@ public function isStatusOk(): bool
     }
 
     /**
-     * @return ContextKey
-     * @phan-override
+     * @todo: Implement this on the API side
+     */
+    public function storeInContext(Context $context): Context
+    {
+        return $context->set(SpanContextKey::instance(), $this);
+    }
+
+    /**
+     * @todo: Implement this on the API side
      */
-    protected static function getContextKey(): ContextKey
+    public function makeCurrent(): Scope
```

Comment:
This is what Java called it, but `makeActive` would might make more sense. Open to other suggestions.

Reply:
I'd have to think back, but I don't think there is a way to make a span current but not active.  `makeActive` feels like the sane choice to me here too.

Reply:
I also thought of `->activate()` if we think that's better 🤷. That seems to be what C# uses.

Reply:
`->activate()` makes sense to me - that feels explicit enough.